Assets/Scripts/Controllers/projectileController.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class projectileController : MonoBehaviour
```

Comment:
`projectileController` -> `ProjectileController`. Classes and Structs should use PascalCase.